include/functions.h

```diff
@@ -79,6 +79,6 @@ void* SysCfb_GetFbEnd(void);
 void RcpUtils_PrintRegisterStatus(void);
 void RcpUtils_Reset(void);
 
-void SystemHeap_Init(void* start, u32 size);
+void RunTime_Init(void* start, u32 size);
```

Comment:
```suggestion
void Runtime_Init(void* start, u32 size);
```
Runtime is typically one word.

Reply:
yeah thats fair. Youll sometimes "C runtime" abbreviated as "crt" so thats what I chose it. But it is typically one word. 
Updated